core/src/com/unciv/models/ruleset/Nation.kt

```diff
@@ -168,7 +168,7 @@ class Nation : INamed {
                 for (unique in originalUnit.uniques.filterNot { it in unit.uniques })
                     textList += "  " + "Lost ability".tr() + "(" + "vs [${originalUnit.name}]".tr() + "): " + unique.tr()
                 for (promotion in unit.promotions.filter { it !in originalUnit.promotions })
-                    textList += "  " + promotion.tr() + " (" + ruleset.unitPromotions[promotion]!!.effect.tr() + ")"
+                    textList += "  " + promotion.tr() + " (" + ruleset.unitPromotions[promotion]!!.uniques.joinToString(",") { it.tr() } + ")"
```

Comment:
This isn't the first place that the difference between Chinese and other languages (in joinToString(",") vs joingToString(", ")  ) appears.
We should probably have a dedicated function like .joinToStringLanguageDependant()